tests/bdd/features/install.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Installing jrnl
+
+    Scenario: Install jrnl with default options
+        Given we use no config
+        When we run "jrnl hello world" and enter
+            |
+            |
```

Comment:
Could you use `\n` for this instead of a pipe? I think that will be a bit more Gherkin-appropriate since the [Gherkin reference](https://cucumber.io/docs/gherkin/reference/) has some allowances for this inside of table cells already.

Reply:
Yeah, I can do that. I can see how that might be confusing.